modules/ROOT/pages/functions/aggregating.adoc

```diff
@@ -300,6 +300,11 @@ count(expression)
 
 |===
 
+[NOTE]
+====
+Neo4j maintains a transactional count store for holding count metadata, which significantly increases the speed of queries using the `count()` function.
+More information about the count store can be read in this link:https://neo4j.com/developer/kb/fast-counts-using-the-count-store/[Neo4j Knowledge Base article].
```

Comment:
```suggestion
For more information about the count store, refer to link:https://neo4j.com/developer/kb/fast-counts-using-the-count-store/[Fast counts using the count store].
```

I think it's always nicer to use the page title, when it doesn't disrupt the style, so as a user I know what I'm gonna get if I click.